src/runtime/debug/debug.go

```diff
@@ -1,6 +1,13 @@
 // Package debug is a dummy package that is not yet implemented.
```

Comment:
Should this comment be removed?

Reply:
it's still kinda dummy - I was wondering about the comment
```go
// Not implemented.
func ReadBuildInfo() 
```
and figured to leave it as it's... only giving an empty one (no modules, no settings etc) except for GoVersion